### PR TITLE
[Bulk] Validation: fixed missing descriptions II 5/30

### PR DESCRIPTION
**Pull request by reemsabaw1**
This PR fixes 29 missing descriptions.

### CODE REVIEW

File: dynamics-nav/SETVIEW-Function--Record-.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "SETVIEW Function (Record)"
+description: SETVIEW Function (Record) Sets the current sort order, key, and filters on a table.
```
Comment (jmart1428) [inline dynamics-nav/SETVIEW-Function--Record-.md:3]: lowercase 's' on sets

File: dynamics-nav/SETVIEW-Function--RecordRef-.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "SETVIEW Function (RecordRef)"
+description: SETVIEW Function (RecordRef) Sets the current sort order, key, and filters on a table.
```
Comment (jmart1428) [inline dynamics-nav/SETVIEW-Function--RecordRef-.md:3]: lowercase 's' on sets

File: dynamics-nav/SETVIEW-Function.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "SETVIEW Function"
+description: SETVIEW Function Sets the current filter view, which defines the sort order, key, and filters, for a record in a filter control on a filter page.
```
Comment (jmart1428) [inline dynamics-nav/SETVIEW-Function.md:3]: lowercase 's' on sets

File: dynamics-nav/SID-Function--Database-.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "SID Function (Database)"
+description: SID Function (Database) Retrieves the security identifier SID of a Windows user account.
```
Comment (jmart1428) [inline dynamics-nav/SID-Function--Database-.md:3]: lowercase 'r' on retrieves

File: dynamics-nav/SKIP-Function--Report--XMLport-.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "SKIP Function (Report, XMLport)"
+description: SKIP Function (Report, XMLport) Skips the current iteration of the current report or XMLport.
```
Comment (jmart1428) [inline dynamics-nav/SKIP-Function--Report--XMLport-.md:3]: lowercase 's' on skips

File: dynamics-nav/SKIPSYSTEMTRIGGERS-Function--Debugger-.md [hunk 6]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "SKIPSYSTEMTRIGGERS Function (Debugger)"
+description: SKIPSYSTEMTRIGGERS Function (Debugger) Enables the debugger to skip code that is inside system triggers.
```
Comment (jmart1428) [inline dynamics-nav/SKIPSYSTEMTRIGGERS-Function--Debugger-.md:3]: lowercase 'e' on enables

File: dynamics-nav/SLEEP-Function.md [hunk 7]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "SLEEP Function"
+description: SLEEP Function Returns control to the operating system for a specified time.
```
Comment (jmart1428) [inline dynamics-nav/SLEEP-Function.md:3]: lowercase 'r' on returns

File: dynamics-nav/Setting-.NET-Framework-Types-to-Target-the-Microsoft-Dynamics-NAV-Windows-Client-or-Server.md [hunk 8]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Setting .NET Framework Types to Target the Microsoft Dynamics NAV Windows Client or Server"
+description: Setting .NET Framework Types to Target the Microsoft Dynamics NAV Windows Client or Server helps extend a navnow object by using .NET Framework interoperability.
```
Comment (jmart1428) [inline dynamics-nav/Setting-.NET-Framework-Types-to-Target-the-Microsoft-Dynamics-NAV-Windows-Client-or-Server.md:3]: Maybe - "Describes how to set the type instance to target the nav_windows (client-side object) or nav_server (server-side object) in Microsoft Dynamics NAV."

File: dynamics-nav/ShowCaption-Property.md [hunk 9]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "ShowCaption Property"
+description: ShowCaption Property Sets whether the text that is specified by the Caption Property is displayed for the control.
```
Comment (jmart1428) [inline dynamics-nav/ShowCaption-Property.md:3]: lowercase 's' on sets

File: dynamics-nav/ShowFilter-Property.md [hunk 10]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "ShowFilter Property"
+description: ShowFilter Property Sets whether the filter pane is shown on a page by default.
```
Comment (jmart1428) [inline dynamics-nav/ShowFilter-Property.md:3]: lowercase 's' on sets

File: dynamics-nav/ShowPrintStatus-Property.md [hunk 11]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "ShowPrintStatus Property"
+description: ShowPrintStatus Property Sets whether a window that shows the printing status of a report when it is run is displayed.
```
Comment (jmart1428) [inline dynamics-nav/ShowPrintStatus-Property.md:3]: lowercase 's' on sets

File: dynamics-nav/SignDisplacement-Property.md [hunk 12]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "SignDisplacement Property"
+description: SignDisplacement Property Sets a value to shift negative values to the right for display purposes only.
```
Comment (jmart1428) [inline dynamics-nav/SignDisplacement-Property.md:3]: lowercase 's' on sets

File: dynamics-nav/SingleInstance-Property.md [hunk 13]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "SingleInstance Property"
+description: SingleInstance Property Sets whether a single instance of the codeunit and codeunit variables are instantiated.
```
Comment (jmart1428) [inline dynamics-nav/SingleInstance-Property.md:3]: lowercase 's' on sets

File: dynamics-nav/Sizing-a-Control-Add-in-on-a-Page.md [hunk 14]
```diff
@@ -12,16 +13,16 @@ caps.latest.revision: 13
 manager: edupont
 ---
 # Sizing a Control Add-in on a Page
-By default, the size of a control add-in on a page is determined by the [System.Drawing.Size](https://docs.microsoft.com/search/index?dataSource=previousVersions&search=System.Drawing.Size) class of the control. The size is static, so when the page is resized in the [!INCLUDE[nav_windows](includes/nav_windows_md.md)], the control add-in size will remain the same.  
+By default, the size of a control add-in on a page is determined by the [System.Drawing.Size](/search/index?dataSource=previousVersions&search=System.Drawing.Size) class of the control. The size is static, so when the page is resized in the [!INCLUDE[nav_windows](includes/nav_windows_md.md)], the control add-in size will remain the same.
```
Comment (jmart1428) [inline dynamics-nav/Sizing-a-Control-Add-in-on-a-Page.md:16]: It looks like all of these links are broken. You may want to get feedback from Derek or Chong on these before submitting it for review.

File: dynamics-nav/Sizing-a-Control-Add-in-on-a-Page.md [hunk 15]
```diff
@@ -12,16 +13,16 @@ caps.latest.revision: 13
...
   
 ## Changing the Size of a Control Add-in  
- To change the size of the control add-in and set it to resize with the page window size, you implement the [WinFormsControlAddInBase.ApplySize](https://docs.microsoft.com/search/index?dataSource=previousVersions&search=Microsoft.Dynamics.Framework.UI.Extensibility.WinForms.WinFormsControlAddInBase.ApplySize(Microsoft.Dynamics.Framework.UI.Extensibility.WinForms.DisplaySize,Microsoft.Dynamics.Framework.UI.Extensibility.WinForms.DisplaySize)) method. This method has the following syntax.  
+ To change the size of the control add-in and set it to resize with the page window size, you implement the [WinFormsControlAddInBase.ApplySize](/search/index?dataSource=previousVersions&search=Microsoft.Dynamics.Framework.UI.Extensibility.WinForms.WinFormsControlAddInBase.ApplySize(Microsoft.Dynamics.Framework.UI.Extensibility.WinForms.DisplaySize,Microsoft.Dynamics.Framework.UI.Extensibility.WinForms.DisplaySize)) method. This method has the following syntax.
```
Comment (jmart1428) [inline dynamics-nav/Sizing-a-Control-Add-in-on-a-Page.md:19]: This one looks broken as well.

File: dynamics-nav/Sizing-a-Control-Add-in-on-a-Page.md [hunk 16]
```diff
@@ -12,16 +13,16 @@ caps.latest.revision: 13
...
+ To change the size of the control add-in and set it to resize with the page window size, you implement the [WinFormsControlAddInBase.ApplySize](/search/index?dataSource=previousVersions&search=Microsoft.Dynamics.Framework.UI.Extensibility.WinForms.WinFormsControlAddInBase.ApplySize(Microsoft.Dynamics.Framework.UI.Extensibility.WinForms.DisplaySize,Microsoft.Dynamics.Framework.UI.Extensibility.WinForms.DisplaySize)) method. This method has the following syntax.  
   
 ```c#  
 public void ApplySize(DisplaySize width, DisplaySize height);  
 ```  
   
- Each dimension is defined by the [DisplaySize](https://docs.microsoft.com/search/index?dataSource=previousVersions&search=Microsoft.Dynamics.Framework.UI.Extensibility.WinForms.DisplaySize) method. This method has the following syntax.  
+ Each dimension is defined by the [DisplaySize](/search/index?dataSource=previousVersions&search=Microsoft.Dynamics.Framework.UI.Extensibility.WinForms.DisplaySize) method. This method has the following syntax.
```
Comment (jmart1428) [inline dynamics-nav/Sizing-a-Control-Add-in-on-a-Page.md:25]: This one looks broken as well.

File: dynamics-nav/Sizing-a-Control-Add-in-on-a-Page.md [hunk 17]
```diff
@@ -55,7 +56,7 @@ this.ApplySize(DisplaySize.Default, new DisplaySize(10, 20, 200));
...
+ You can use the [WinFormsControlAddInBase.ApplySize](/search/index?dataSource=previousVersions&search=Microsoft.Dynamics.Framework.UI.Extensibility.WinForms.WinFormsControlAddInBase.ApplySize(Microsoft.Dynamics.Framework.UI.Extensibility.WinForms.DisplaySize,Microsoft.Dynamics.Framework.UI.Extensibility.WinForms.DisplaySize)) method multiple times in the control add-in code to change the width and height. However, you should use it as part of the [CreateControl](/search/index?dataSource=previousVersions&search=Microsoft.Dynamics.Framework.UI.Extensibility.WinForms.WinFormsControlAddInBase.CreateControl) method.
```
Comment (jmart1428) [inline dynamics-nav/Sizing-a-Control-Add-in-on-a-Page.md:59]: This one looks broken as well.

**Comment by reemsabaw1** [comment]
@buck1ey 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Pull request merged**